frontend/src/interfaces/pagedItems.tsx

```diff
@@ -1,7 +1,7 @@
 /**
- * A PagedItem contains a page of items for pageing.
+ * A PagedItem contains a page of items for paging.
  */
-export interface IPagedItems<T> {
+export interface IPagedItems<T extends object = {}> {
```

Comment:
I don't understand this?

Reply:
if you use it as `IPagedItems` (no generic type supplied) it will assume `object`. Before it would error out and ask for a specific type. It allows for more flexibility - we can change at any time